src/main/java/com/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -42,6 +42,7 @@ public ResponseEntity<Object> handleRegistrationException(RegistrationException
     @ExceptionHandler(Exception.class)
     public ResponseEntity<Object> handleException(Exception ex) {
         Map<String, Object> error = new HashMap<>();
+        error.put("exceptionType", ex.getClass().getName());
```

Comment:
why did you add changes not related to this HW to this PR? Let's do it in a separate PR and do not send them on review, if it's related to the bugfix/refactor etc

Reply:
not fixed

Reply:
> not fixed

Since we do not have a separate task for it and the change is minimal I decided to put it into this PR, but it does not have any real impact on the way the program runs, so I can also remove this line completely